include/swift/Frontend/Frontend.h

```diff
@@ -413,9 +431,26 @@ class CompilerInstance {
     return Invocation.getFrontendOptions().EnableSourceImport;
   }
 
+  /// Gets the set of SourceFiles which are the primary inputs for this
+  /// CompilerInstance.
+  const llvm::SetVector<SourceFile*> &getPrimarySourceFiles() {
```

Comment:
…and then this can just be an ArrayRef, if we don't care about its set-ness anymore.

Reply:
Done.